README.md

```diff
@@ -1 +1,93 @@
-TODO.
+# Kubewarden policy volumemounts-policy
+
+## Description
+
+Policy that inspects containers, init containers, and ephemeral containers, and
+restricts their usage of volumes by  checking the `volume` name being used in
+`volumeMounts[*].name`.
+
+The policy can either target `Pods`, or [workload
+resources](https://kubernetes.io/docs/concepts/workloads/) (`Deployments`,
+`ReplicaSets`, `DaemonSets`, `ReplicationControllers`, `Jobs`, `CronJobs`) by
+setting the policy's `spec.rules` accordingly.
+
+Both have trade-offs:
+* Policy targets Pods: Different kind of resources (be them native or CRDs) can
+  create Pods. By having the policy target Pods, we guarantee that all the Pods
+  are going to be compliant, even those created from CRDs.
+  However, this could lead to confusion among users, as high level Kubernetes
+  resources would be successfully created, but they would stay in a non
+  reconciled state. Example: a Deployment creating a non-compliant Pod would be
+  created, but it would never have all its replicas running.
+* Policy targets workload resources (e.g: Deployment): the policy inspect higher
+  order resource (e.g. Deployment): users will get immediate feedback about
+  rejections.
+  However, non compliant pods created by another high level resource (be it
+  native to Kubernetes, or a CRD), may not get rejected.
+
+
+## Settings
+```yaml
+reject: anyIn # one of anyIn (default, denylist), anyNotIn (allowlist), allIn, allNotIn
+volumeMountNames:  # list of volumeMounts.name to match using the defined reject operator
+  - foo
+  - bar
+  - baz
+```
+
+- `allIn` checks if all of the volumeMountNames are in the Pod/Workload resource
+- `anyIn` checks if any of the volumeMountNames are in the Pod/Workload resource
+- `allNotIn` checks if all of the volumeMountNames are not in the Pod/Workload resource
+- `anyNotIn` checks if any of the volumeMountNames are not in the Pod/Workload resource
+
+## Examples
+
+```yaml
+# denylist, reject volumeMounts named `my-volume` or `my-volume2`
+reject: anyIn
+volumeMountsNames:
+  - my-volume
+  - my-volume2
+```
+
+```yaml
+# allowlist, only allow volumeMounts named `my-volume3` or `my-volume4`
+reject: anyNotIn
+volumeMounts:
+  - my-volume3
+  - my-volume4
+```
```

Comment:
I read it as:

> **reject** **any** volume mount name that **does not** belong to this list `[ my-volume3, my-volume4]`

This is clear to me

Reply:
Resolving, I will leave also this one untouched.